src/ledger/onChainQuerier.ts

```diff
@@ -0,0 +1,6 @@
+export interface OnChainQuerier {
+	queryTokensOwnedByAddress(
+		token: string,
+		address: string,
+	): Promise<string[]>;
```

Comment:
I propose to return `Promise<Tokens>`, since it is part of our API and it makes clear what to expect.